requirements-dev.txt

```diff
@@ -0,0 +1,12 @@
+pynwb
```

Comment:
why have a different requirements-dev file here? The constraints on ipywidgets is important

Reply:
this is old, I don't remember why... but we can safely remove it